ts/store/reducers/wallet/transactions.ts

```diff
@@ -0,0 +1,93 @@
+/**
```

Comment:
Need to store also the fetching status (loading/error). If you use a proper action type naming you get the loading/error status for free. Look at:

https://github.com/teamdigitale/italia-app/blob/master/ts/store/reducers/loading.ts
https://github.com/teamdigitale/italia-app/blob/master/ts/store/reducers/error.ts